packages/playwright-core/src/server/isomorphic/utilityScriptSerializers.ts

```diff
@@ -18,6 +18,7 @@ export type SerializedValue =
     undefined | boolean | number | string |
     { v: 'null' | 'undefined' | 'NaN' | 'Infinity' | '-Infinity' | '-0' } |
     { d: string } |
+    { u: string } |
```

Comment:
I wrote this file's changes myself -- is there a script other than `npm run build` to generate them?

Reply:
Yep, this should be enough. Usually, passing tests means everything is good 😄